src/Test.UnitTests.BinSkim.Driver/CompilerDataLoggerTests.cs

```diff
@@ -788,7 +788,7 @@ public static string GetExampleSarifPath(Sarif.SarifVersion sarifVersion)
 
         internal static string GetTestDirectory(string relativeDirectory)
         {
-            var codeBaseUrl = new Uri(Assembly.GetExecutingAssembly().CodeBase);
+            var codeBaseUrl = new Uri(Assembly.GetExecutingAssembly().Location);
             string codeBasePath = Uri.UnescapeDataString(codeBaseUrl.AbsolutePath);
```

Comment:
Assembly.CodeBase Property is obsolete, so change it to Assembly.Location.

Reply:
OK, here's the info I asked for. :)